SPECS/kernel/config

```diff
@@ -221,7 +229,7 @@ CONFIG_RD_XZ=y
 CONFIG_RD_LZO=y
 CONFIG_RD_LZ4=y
 CONFIG_RD_ZSTD=y
-# CONFIG_BOOT_CONFIG is not set
+CONFIG_BOOT_CONFIG=y
```

Comment:
Allows you to pass in a boot config

https://cateee.net/lkddb/web-lkddb/BOOT_CONFIG.html